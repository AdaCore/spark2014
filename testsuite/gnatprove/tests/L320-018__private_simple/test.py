from test_support import *

prove() # actual flow error, to be fixed in M909-012
