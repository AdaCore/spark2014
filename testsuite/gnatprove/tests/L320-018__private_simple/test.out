q.adb:10:4: info: assertion proved
